environment.yml

```diff
@@ -3,12 +3,12 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - python==3.8
-  - pandas==1.5.3
-  - numpy<2
-  - xarray
+  - python
+  - pandas
+  - numpy
+  - xarray>=2025.01.2
```

Comment:
this one needs to be like that, otherwise it's not compatible with the newest zarr package

Reply:
">" for a version is not a problem, "<" is -> ok for me
please also set it in [setup.cfg ](https://github.com/awst-austria/qa4sm-preprocessing/blob/master/setup.cfg#L60)

Reply:
done